.github/filters/ci.yml

```diff
@@ -43,10 +43,10 @@ new-client-dependencies-project: &new-client-dependencies-project
   - oxidation/client/tsconfig.json
 
 web: &web
-  - oxidation/client/public
-  - oxidation/client/resources
-  - oxidation/client/src
-  - oxidation/client/tests
+  - oxidation/client/public/**
+  - oxidation/client/resources/**
+  - oxidation/client/src/**
+  - oxidation/client/tests/**
   - oxidation/client/*.js
```

Comment:
Is it normal we only take into consideration .js/.ts in the root folder ? I'd say we want everything (especially given `ionic.config.json` exists ^^)

Reply:
We don't want everything given we store the code for `android`, `ios` & `electron` (changing the code for one of those don't require to run the web tests)

Reply:
I've changed the pattern for `oxidation/client/*.{ts,js,json}`